engine-comparison/dispatcher.sh

```diff
@@ -66,6 +66,17 @@ build_benchmark_using() {
   mkdir $SEND_DIR
 
   cp ${BUILDING_DIR}/${BENCHMARK}-${FUZZING_ENGINE} $SEND_DIR
+
+  #TODO: make these gsutil cp?
```

Comment:
Yeah, it seems to be:

> src_url must specify a directory, bucket, or bucket subdirectory

https://cloud.google.com/storage/docs/gsutil/commands/rsync